adafruit_platformdetect/board.py

```diff
@@ -752,6 +752,25 @@ def any_libre_computer_board(self) -> bool:
         """Check whether the current board is any defined Libre Computer board."""
         return self.id in boards._LIBRE_COMPUTER_IDS
 
+    @property
+    def os_environ_board(self) -> bool:
+        """Check whether the current board is an OS environment variable special case."""
+
+        def lazily_generate_conditions():
+            yield self.board.FTDI_FT232H
+            yield self.board.FTDI_FT2232H
+            yield self.board.MICROCHIP_MCP2221
+            yield self.board.BINHO_NOVA
+            yield self.board.GREATFET_ONE
+            yield self.board.PICO_U2IF
+            yield self.board.FEATHER_U2IF
+            yield self.board.ITSYBITY_U2IF
+            yield self.board.MACROPAD_U2IF
+            yield self.board.QTPY_U2IF
+            yield self.board.QT2040_TRINKEY_U2IF
```

Comment:
Is there any benefit to a generator here?

Reply:
Mostly for readability, but also if it finds one, it stops checking.